README.rst

```diff
@@ -0,0 +1,45 @@
+###########################
+Jupyter-OpenModelica Kernel
+###########################
+
+An OpenModelica Kernel for Jupyter Notebook.
+All commands are interpreted by OMPython which communicates with OpenModelica Compiler
+and the result is presented to user.
+
+Dependencies
+============
+The following software has to be installed:
+
+- `Python <https://www.python.org/downloads/>`_
+- `Jupyter Notebook <https://jupyter.readthedocs.org/en/latest/install.html>`_
+- `OpenModelica <https://openmodelica.org>`_ v1.12.0 or newer
+- `OMPython <https://github.com/OpenModelica/OMPython>`_.
+
+When using an existing Python installation, it is strongly recommended to update ``pip`` and ``jupyter`` by running::
+
+  python -m pip install -U pip setuptools wheel
+  python -m pip install -U jupyter
```

Comment:
Why the update of setuptools and wheel ?

Reply:
To make sure numpy is installed from a wheel and not compiled from source.

Reply:
also see https://github.com/travis-ci/travis-ci/issues/2650#issuecomment-252285413